src/_main_/settings.py

```diff
@@ -236,7 +237,9 @@
 # Static files (CSS, JavaScript, Images)
 STATIC_URL = '/static/'
 MEDIA_URL = '/media/'
+MEDIA_ROOT = os.path.join(BASE_DIR, 'media')
 
+STATIC_ROOT = os.path.join(BASE_DIR, 'static')
```

Comment:
Do these definitions of MEDIA_ROOT and STATIC_ROOT change where those files get deployed and could the change screw things up?

Reply:
Based on the resources online that I looked at, the answer is no.